src/widget/form/genericchatform.cpp

```diff
@@ -199,6 +203,7 @@ GenericChatForm::GenericChatForm(QWidget *parent)
             this, &GenericChatForm::onChatContextMenuRequested);
 
     new QShortcut(Qt::CTRL + Qt::SHIFT + Qt::Key_L, this, SLOT(clearChatArea()));
+    new QShortcut(Qt::ALT + Qt::Key_Q, this, SLOT(quoteSelectedText()));
```

Comment:
@SkyzohKey As for me, the way I implemented it is more intuitive. I use it in other old instant messengers.
Anyway, your suggestion may be implemented as radio buttons in qTox settings: "Insert quotation to message input area" (by default) or "Copy quotation to clipboard". On the other hand, it makes current simple implementation more complicated.